views/trust-information.html

```diff
@@ -0,0 +1,105 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Add trust information for the beneficial owners – {{ serviceName }} – GOV.UK
+{% endblock %}
+
+{% block beforeContent %}
```

Comment:
need to change the `block beforeContent ` to `block backLink`
when you navigate through the page you'll notice the 'Alpha' banner dissappears on the new page as you are overriding the block beforeContent in layout.html. Just need to override the block backLink instead